src/components/DropDown.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Person } from '../types/Person';
+type Props = {
+  visiblePeople: Person[];
+  appliedQuery: string;
+  onSelected: (value: Person) => void;
+};
+export const DropDown: React.FC<Props> = ({
+  visiblePeople,
+  appliedQuery: query,
+  onSelected: onSelected,
+}) => {
+  return (
+    <div className="dropdown-menu" role="menu" data-cy="suggestions-list">
+      <div className="dropdown-content">
+        {query &&
```

Comment:
The conditional rendering based on `query` might cause the dropdown to not display any items if `query` is an empty string. Ensure this behavior is intended, or consider revising the condition to better match the desired functionality.